src/terms/xc.jl

```diff
@@ -454,3 +517,55 @@ function divergence_real(operand, basis)
     # TODO: forcing real-valued ifft; should be enforced at creation of array
     irfft(basis, gradsum; check=Val(false))
 end
+
+"""
+Compute the form factors (Fourier transforms of the core charge density) at all unique
+`|G|` in `Gvecs_cart` for an atom centered at 0.
+"""
+function core_density_form_factors(el::Element, qs::AbstractArray{T})::IdDict{T,T} where {T}
```

Comment:
These functions should probably move somewhere else. Rename guess_density to atomic_densities and put that whole lot there?

Reply:
Agree this feels out of place here.